src/themes/brutalist.css

```
@@ -0,0 +1,832 @@
+   */
+  --wa-z-index-drawer: 700;
+  --wa-z-index-dialog: 800;
+  --wa-z-index-dropdown: 900;
+  --wa-z-index-alert-group: 950;
+  --wa-z-index-tooltip: 1000;
+
+  /**
+   * Transitions
+   */
+  --wa-transition-normal: 250ms;
+  --wa-transition-fast: 150ms;
+  --wa-transition-faster: 50ms;
+
+  /**
+   * Component groups
+   */
+  --wa-form-controls-background: var(--wa-color-surface-default);
+
+  --wa-form-controls-corners: var(--wa-corners-s);
+
+  --wa-form-controls-activated-color: var(--wa-color-neutral-spot);
+  --wa-form-controls-resting-color: var(--wa-color-neutral-border-subtle);
+
+  --wa-form-controls-label-color: var(--wa-color-text-normal);
+  --wa-form-controls-label-font-weight: var(--wa-font-weight-normal);
+  --wa-form-controls-label-line-height: var(--wa-font-line-height-regular);
+
+  --wa-form-controls-value-color: var(--wa-color-text-normal);
+  --wa-form-controls-value-font-weight: var(--wa-font-weight-body);
+  --wa-form-controls-value-line-height: var(--wa-font-line-height-compact);
+
+  --wa-form-controls-placeholder-color: var(--wa-color-base-60);
+
+  --wa-form-controls-height-s: calc(
+    var(--wa-space-xs) * 2 + var(--wa-font-size-s) * var(--wa-form-controls-value-line-height)
+  );
+  --wa-form-controls-height-m: calc(
+    var(--wa-space-s) * 2 + var(--wa-font-size-m) * var(--wa-form-controls-value-line-height)
+  );
+  --wa-form-controls-height-l: calc(
+    var(--wa-space-m) * 2 + var(--wa-font-size-l) * var(--wa-form-controls-value-line-height)
+  );
+
+  --wa-form-controls-required-content: '*';
+  --wa-form-controls-required-content-color: inherit;
+  --wa-form-controls-required-content-offset: -0.1em;
+
+  --wa-panel-corners: var(--wa-corners-m);
+
+  --wa-flow-spacing: 1.5rem;
+
+  /**
+   * From 2.x
+   */
+  --wa-tooltip-arrow-size: 0.375rem;
+
+  /* premium custom settings - general and light */
+
+  pre {
+    border-left: var(--wa-border-width-m) var(--wa-panel-border-style) var(--wa-color-neutral-border-subtle);
+  }
+
+  .textmarker-effect {
+    position: relative;
+    padding-left: 10px;
+    padding-right: 10px;
+
+    &:before {
+      position: absolute;
+      z-index: -1;
+      content: '';
+      background: #fbff2b;
+      height: 20px;
+      left: 0;
+      bottom: -10px;
+      width: 0%;
+      opacity: 0.7;
+      transition: all 0.5s;
+    }
+
+    &:hover:before {
+      width: 100%;
+    }
+  }
+
+  p a {
+    position: relative;
+    --wa-link-decoration-default: none;
+    --wa-link-decoration-hover: none;
+
+    &:before {
+      position: absolute;
+      z-index: -1;
+      content: '';
+      background: var(--wa-color-yellow-80);
+      width: 100%;
+      bottom: 0;
+      height: 50%;
+      opacity: 0.7;
+      transition: all var(--wa-transition-faster) ease-in-out;
+    }
+
+    &:hover:before {
+      height: 100%;
+    }
+  }
+
+  wa-card::part(image) {
+    border-bottom: var(--wa-panel-border-width) var(--wa-panel-border-style) var(--wa-color-surface-border);
```

```suggestion
    border-bottom: var(--border-width) var(--border-style) var(--border-color);
```

Totally optional, but we could leverage the `wa-card` custom properties here to ensure this border always matches the rest of the card's borders.